### PR TITLE
fix: switch back to main release-please project

### LINKED ISSUE #109: Switch back to main dist of release-please
Release please ended up merging [my PR](https://github.com/googleapis/release-please/pull/2323)!

This means that we can ditch my fork of the project and switch back to using release please v16.12.1 or higher.

### PR DESCRIPTION
**Pull request by stdavis**
Closes #109

### CODE REVIEW

**Comment by jacobdadams** [comment]
nevermind, saw the linked issue.

File: action.yml [hunk 1]
```diff
@@ -118,18 +118,14 @@ runs:
     - name: Set up Release Please
       shell: bash
       run: |
-        git clone https://github.com/stdavis/release-please
-        cd release-please
-        git switch stable-fix
-        echo "installing dependencies"
-        npm install
+        npm i release-please -g
```
Comment (steveoh) [inline action.yml:121]: do we want to add this to a package json or are we ok with not having a static version?

**Review by steveoh — APPROVED** [review]
do we wa

File: action.yml (hunk 1 above)
Comment (stdavis) [reply action.yml:121]: Hmm. Good point. There is no package.json file for this project. But I could pin to a major version with the install command. I'll give it a try...

**Review by steveoh — APPROVED** [review]

**Pull request merged**